src/main/resources/application.properties

```diff
@@ -1 +1,8 @@
+spring.datasource.url=jdbc:mysql://localhost:3306/ticket_app?serverTimezone=UTC
+spring.datasource.username=root
+spring.datasource.password=Rostuk111@
```

Comment:
Don't commit and push your password. It is very dangerous to publish your password in a public repository.